src/api_server/src/session.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022 by Rivos Inc.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use attestation_service::types::AttestationResults;
+use kbs_types::{Request, Tee};
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex, RwLock};
+use std::time::{Duration, Instant};
+use uuid::Uuid;
+
+static SESSION_TIMEOUT: u64 = 5 * 60;
```

Comment:
Do we want to set this parameter as configurable rather than hard coded?

Reply:
Yes, absolutely. I will add a CLI in an additional commit, this is still WIP ;-)